thorlcr/graph/thgraph.hpp

```diff
@@ -121,14 +121,23 @@ interface IThorResult : extends IInterface
     virtual const void * getLinkedRowResult() = 0;
 };
 
+enum ThorGraphResultType:unsigned
+{
+    thorgraphresult_nul = 0x00,
+    thorgraphresult_distributed = 0x01,
+    thorgraphresult_allownulls=0x02,
```

Comment:
I don't use allownulls, not sure it's useful.
I'll remove and address related issues.